pkg/operator/ceph/file/mds/mds_liveness_probe.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+ROOK_CEPH_MON_HOST="{{ .RookCephMonHost }}"
+ROOK_CEPH_MON_INITIAL_MEMBERS="{{ .RookCephMonInitialMembers }}"
+DAEMON_MDS="{{ .DaemonMds }}"
+FILESYSTEM_NAME="{{ .FilesystemName }}"
+
+outp="$(ceph fs dump --mon-host="$ROOK_CEPH_MON_HOST" --mon-initial-members="$ROOK_CEPH_MON_INITIAL_MEMBERS" --format json 2>&1)"
+rc=$?
+if [ $rc -ne 0 ]; then
+    echo "ceph mds dump check failed with the following output:"
+    echo "$outp" | sed -e 's/^/> /g'
+    echo "we couldn't get the information to reliably tell if the MDS should be restarted or not"
```

Comment:
Because this behavior may be surprising to some users, let's be very clear in messages what the probe is returning, what purpose is in returning the result, and what result it has gotten to make that decision.

And also avoid "we" in messages.

```suggestion
    echo "passing probe to avoid restarting MDS. cannot determine if MDS is unhealthy. restarting MDS risks destabilizing ceph/filesystem, which is likely unreachable or in error state"
```